### ISSUE TITLE
Weird vertical line of pixels on spawned text bubbles

### ISSUE THREAD

**Issue by gfodor** [mozilla/hubs#1569]
**Describe the bug**
If you spawn a text bubble, it looks like you can see past the edge of the left side and there's a vertical bar of black pixels:

**To Reproduce**
Create a room, and spawn a label.

**Expected behavior**
There should be no black pixels.

**Screenshots**

![image](https://user-images.githubusercontent.com/220020/61844719-408dd180-ae55-11e9-8b3d-50d83da79395.png)

**Desktop (please complete the following information):**
Windows Chrome

**Comment by robertlong** [comment]
This is likely due to a combination of anisotropic filtering and our texture atlas solution. The fix is likely to add more padding to the various mipmaps.

**Comment by netpro2k** [comment]
Yep this is definitely an atlas bleeding issue. I fixed it for miplevel 0 but still happens on higher mip levels. Its technically not limited to just chat bubbles and can happen with anything batched, but its just easier to notice because of the alpha cutout on chat messages.

**Comment by netpro2k** [comment]
Since batching is disabled by default now indefinitely, bumping this down to p3

**Comment by joshmarinacci** [comment]
Still happening?

**Comment by robertlong** [comment]
Only if we enable batching for images again. The solution for this is as I described above, we need a little more padding between images in the texture atlas.
